services/slider.js

```diff
@@ -48,11 +48,13 @@ const createSlider = async (req) => {
 const deleteSlider = async (req) => {
     if (!mongoose.Types.ObjectId.isValid(req.params.id)) throw NSErrors.UnprocessableEntity;
     const doc = await Slider.findOneAndRemove({_id: req.params.id});
+    if (!doc) {
+        throw NSErrors.NotFound;
+    }
```

Comment:
⚠️ change the service

-> no custom error ? (like `NSErrors.NotFound;`)

Reply:
You should create one